src/views/CartoonPage.vue

```diff
@@ -0,0 +1,150 @@
+    <div class="col1" data-aos="slide-right" data-aos-duration="1500">
+      <p>
+        Nav (pseudocode)
+        <br />
+        for each item in the trivia array, loop to display the nav link
+        <br />
+        on click of the navlink, dynamically render the content of trivia card
+        <br />
+        bootstrap reusable tab component? jQuery show/hide divs?
+      </p>
+    </div>
+
+    <div class="col2" data-aos="zoom-in" data-aos-duration="1500">
+      <div>
+        <div v-if="validKey">
+          <h1>
+            <b>{{ data.title.toUpperCase() }}</b>
+          </h1>
+          <video controls>
+            <source :src="video_url" :alt="data.title" type="video/mp4" />
+            <source :src="video_url" :alt="data.title" type="video/avi" />
+          </video>
+        </div>
+        <div v-else>
+          Url /{{ $route.params.key }} is not existant
+          {{ validKey }}
+        </div>
+      </div>
+      <div>
+        <p>
+          <b> //TODO: find in bootstrap library for a card component </b>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellendus,
+          molestiae architecto! Quisquam explicabo quaerat, vitae quis vel velit
+          voluptate! Repellendus sint quibusdam aliquam debitis saepe
+          exercitationem, quia optio sit minus consequuntur. Obcaecati numquam
+          neque maiores nulla pariatur quo nisi expedita debitis sed, vero sit
+          eum. Quidem ratione ipsa eligendi illo inventore neque! Aperiam quasi
+          rem porro aliquid illo, autem modi tempore eius voluptate! At
+          voluptatem deleniti exercitationem sunt provident quos maiores.
+          Excepturi beatae laboriosam eveniet iure, enim harum aspernatur porro
+          laudantium laborum eius error quas? Libero beatae delectus aliquid
+          incidunt vero necessitatibus eius amet itaque magnam iste maiores,
+          voluptatibus culpa.
+        </p>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+//import cartoonData
+import cartoonData from "../data/featuredCartoon.json";
+
+export default {
+  name: "CartoonPage",
+  computed: {
+    validKey: function () {
+      for (let i = 0; i < cartoonData.length; i++) {
+        const d = cartoonData[i];
+        if (d.route === this.$route.params.key) {
+          return true;
+        }
+      }
+      return false;
+    },
+    data: function () {
+      for (let i = 0; i < cartoonData.length; i++) {
+        const d = cartoonData[i];
+        if (d.route === this.$route.params.key) {
+          return d;
+        }
+      }
+      return 0;
+    },
+    video_url: function () {
+      let url = this.data.video;
+      try {
+        url = require("@/assets/" + url); // match the url and use that image
+      } catch (e) {
+        url = require("@/assets/default.jpg"); // use a default image if error
```

Comment:
better return an empty video url instead of having media decoder to fetch the file then fails since the video element always fail with image file

Reply:
how to "return empty url" (I console log the error instead)

Reply:
just return empty string since in html empty string is null